### PR TITLE
close submenu after route change

### PR DESCRIPTION
**Pull request by jvonasek**

### CODE REVIEW

File: src/components/Layout/Header/menu/HeaderSubMenu.tsx [hunk 1]
```diff
@@ -30,6 +35,14 @@ export const HeaderSubMenu = ({ item }: Props) => {
   const { key, subItems } = item
   const isActive = subItems.some(({ href }) => match({ to: href }))
 
+  const {
+    current: { pathname },
+  } = useLocation()
+
+  useEffect(() => {
```
Comment (vkulinich-cl) [inline src/components/Layout/Header/menu/HeaderSubMenu.tsx:42]: Wouldn't it better to add onClick handler to the Link when you click it, just close the popup?

Comment (kolebjak) [reply src/components/Layout/Header/menu/HeaderSubMenu.tsx:42]: I agree. This also won't solve a case when user clicks on a menu item that links to the current page.

Comment (jvonasek) [reply src/components/Layout/Header/menu/HeaderSubMenu.tsx]: updated

**Review by kolebjak — APPROVED** [review]
lgtm

**Review by vkulinich-cl — APPROVED** [review]

**Pull request merged**